frontend/store/playbackManager.ts

```diff
@@ -98,6 +98,13 @@ export const playbackManagerStore = defineStore('playbackManager', {
       queue.push(...translatedItem);
       this.queue = queue;
     },
+    async removeFromQueue(item: BaseItemDto) {
+      const queue = Array.from(this.queue);
+      const translatedItem = await this.translateItemsForPlayback(item);
+
+      queue.splice(queue.indexOf(translatedItem[0]), 1);
+      this.queue = queue;
+    },
```

Comment:
For making this function more modular, I think it would be a nice touch to accept the item.Id directly. If we have the BaseItemDto in the calling code, it's as easy as doing ``removeFromQueue(item.Id)``. But if we have the id as an string, we need to fetch the item to call the function, so TypeScript doesn't complain.

```suggestion
    removeFromQueue(itemId: string) {
      if (this.queue.includes(itemId)) {
          const queue = Array.from(this.queue);

          queue.splice(queue.indexOf(item, 1);
          this.queue = queue;
      }
    },
```

Remember to update the call in DraggableQueue accordingly